firecrown/ccl/sources/tests/test_cl_source.py

```diff
@@ -0,0 +1,28 @@
+from ..sources import ClusterSource
+import os
+bin_data = {
+    'z_min':0.1,
+    'z_max':1.0,
+    'proxy_min':20,
+    'proxy_max':40,
+    'area_eff':10,
+}
+
+f = open('dummy_cl_bin_data.csv', 'w')
+print(','.join(bin_data.keys())+'\n'+
+        ','.join([str(i) for i in bin_data.values()]),
+        file=f)
+f.close()
+cl = ClusterSource('dummy_cl_bin_data.csv')
+os.system('rm dummy_cl_bin_data.csv')
+print(cl)
```

Comment:
This needs to be done in a temporary directory made within the test. See `tempfile.TemporaryDirectory()` and please use a context manager.